Gaijinizer patch/Map038.txt

```diff
@@ -135,38 +136,38 @@ would join M.O.W.#MTLed
 だが、当時俺たち二人は軍で活動していて、レイナに会う事も
 次第に少なくなっていたんだ。
 > CONTEXT: Map038/events/4/pages/0/75/Dialogue/yuuma
-Of course, I and Micah fought against.
-But at the time we both were active in the
-military, and we would meet Reina
-It was getting less and less.#MTLed
+Naturally, Micah and I were strongly against it.
+But at the time we were both occupied with our
+military duties, and there were fewer and fewer
+opportunities to meet her.#Ron
 > END STRING
 
 > BEGIN STRING
 そして数週間もしないうちに、俺たちの心配をよそに
 
 彼女はこつぜんと、姿を消した…。
 > CONTEXT: Map038/events/4/pages/0/80/Dialogue/yuuma
-And in a couple of weeks, let's not worry about
-us. She has disappeared, ...#MTLed
+And so within a few weeks, she'd ignored our
+concerns and disappeared...#Ron
 > END STRING
 
 > BEGIN STRING
 不定期ではあるが、色んな形で便りは届いている。
 
 おそらく、内部の目を盗んで送ってきているんだろう。
 > CONTEXT: Map038/events/4/pages/0/86/Dialogue/yuuma
-Although irregular, the letter has arrived in
-various forms. Perhaps they're stealing inside
-and sending.#MTLed
+Although it's irregular, we do hear from her in a
+variety of ways. She's probably communicating with
+us without them realising.#Ron
```

Comment:
Rephrasing:
>_**The schedule is irregular, but we get messages from her
in a variety of ways. She's probably covertly communicating
with us right under M.O.W's watchful eye.**_
(This isn't a true direct translation, feel free to pare it down.)

Reply:
Altered a bit.

```
The schedule is irregular, but we get messages
from her in a variety of ways. They probably don't
realise she's been covertly communicating with
us.
```